packages/story-editor/src/components/publishModal/mainContent/checklistButton.js

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import { Icons } from '@googleforcreators/design-system';
+import { __ } from '@googleforcreators/i18n';
+/**
+ * Internal dependencies
+ */
+import { ISSUE_TYPES } from '../../checklist/constants';
+import {
+  useCategoryCount,
+  useHasChecklist,
+} from '../../checklist/countContext';
+import { ToggleButton } from '../../toggleButton';
+
+const ChecklistButton = ({ handleReviewChecklist }) => {
+  const hasChecklist = useHasChecklist();
+  if (!hasChecklist) {
+    return null;
```

Comment:
💭 Could use the hook inside `MainContent` and conditionally render the button there? What do you think about having the hook there vs here?

Reply:
yes, i actually just pushed that up after brooke pointed out my linting issue. thanks!